.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - feature/cek/release-workflow
+    tags:
+      - "*"
+
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout commit
+        uses: actions/checkout@v2
+
+      - name: Install correct version of Node
+        shell: bash -l {0}
+        run: nvm install
+
+      - name: Set up registry authentication
+        uses: actions/setup-node@v1
+        with:
+          registry-url: https://registry.npmjs.org
```

Comment:
This sets up an `.npmrc` file to assist in authenticating to npm. By excluding `node_version`, we defer to whichever version of Node is in `PATH`, which will be managed by `nvm` here.

Reply:
Similar comment here regarding use of `nvm` vs. allowing `setup-node` to do its thing with a lose pin on the LTS (e.g., 12.x).